tests/migration/namespace.go

```diff
@@ -561,20 +563,35 @@ var _ = Describe(SIG("Live Migration across namespaces", decorators.RequiresDece
 				checkEFI(targetVMI)
 			})
 		})
+	})
 
-		DescribeTable("should be able to cancel a migration by deleting the migration resource", decorators.SigStorage, func(deleteSource bool) {
-			const timeout = 180
-			migrationID := fmt.Sprintf("mig-%s", rand.String(5))
-			sourceVMI := libvmifact.NewCirros(
+	Context("with migration policy", func() {
+		var (
+			sourceVMI, targetVMI *v1.VirtualMachineInstance
+			policy               *migrationsv1.MigrationPolicy
+		)
+
+		BeforeEach(func() {
+			sourceVMI = libvmifact.NewCirros(
 				libvmi.WithNamespace(testsuite.NamespaceTestDefault),
 				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
 				libvmi.WithNetwork(v1.DefaultPodNetwork()),
 			)
-			targetVMI := sourceVMI.DeepCopy()
+			By("limiting the bandwidth of migrations")
+			policy = CreateMigrationPolicy(virtClient, PreparePolicyAndVMIWithBandwidthLimitation(sourceVMI, resource.MustParse("1Ki")))
+
+			targetVMI = sourceVMI.DeepCopy()
 			targetVMI.Namespace = testsuite.NamespaceTestAlternative
+		})
 
-			By("limiting the bandwidth of migrations in the test namespace")
-			CreateMigrationPolicy(virtClient, PreparePolicyAndVMIWithBandwidthLimitation(sourceVMI, resource.MustParse("1Ki")))
+		AfterEach(func() {
+			By("deleting the migration policy")
+			DeleteMigrationPolicy(virtClient, policy)
```

Comment:
If other tests don't delete their migration policy, I bet it's because it's auto-deleted by the test suite, no need to be redundant (just like we don't delete VMs/VMIs)